### ISSUE TITLE
Getting error using the default code sample

### ISSUE THREAD

**Issue by fbritoferreira** [evertdespiegeleer/zhttp#14]
Hey 👋 
Im getting a import / require error when I try to run the default code snippet in the readme, could be nothing but wanted to flag it to see if you are also facing these issues

```sh
index.js:25
var import_core = require("@zhttp/core");
                  ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /node_modules/.pnpm/@zhttp+core@0.0.6_@zhttp+errors@0.0.6_zod@3.22.4/node_modules/@zhttp/core/dist/main.js from index.js not supported.
Instead change the require of main.js in index.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (index.js:25:19) {
  code: 'ERR_REQUIRE_ESM'
}

Node.js v20.11.0
   ```

**Comment by evertdespiegeleer** [comment]
Hey, thanks for the issue!
Ah – ESM, such a pain sometimes 😅
I just setup quick npm project with only zod and zhttp/core, copied the exact example code and it seems to work fine for me.
However, I did notice that I get a similar error when I switch from `require`ing rather than `import`ing. I'll look into it.
Are you using commonjs as well?

**Comment by evertdespiegeleer** [comment]
It should be fixed in 0.0.7. Would you be willing to validate wether your issue is fixes?

**Comment by fbritoferreira** [comment]
@evertdespiegeleer It works fine now
